install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -741,10 +741,12 @@ label.
 
 This means two things for unprivileged containers:
 
-- The volume will be given a `type` which is accessible by unprivileged
-containers. This type is usually `container_file_t`, which treats volumes as container content. Previously, the label specified was `svirt_sandbox_file_t`. This label
-is no longer used due to security concerns.
-- If a `level` is specified, the volume will be labeled with the given MCS
+- The volume is given a type that is accessible by unprivileged containers.
+This type is usually `container_file_t` in RHEL 7.5+, which treats volumes as
+container content. In previous RHEL versions, RHEL 7.4, 7.3, and so forth, the label
```

Comment:
Maybe "If you use an earlier version of RHEL 7, specify \`svirt_sandbox_file_t`."

Reply:
@kalexand-rh I believe that the container is given a type. I changed the wording here.

Reply:
Thank you. :)